README.md

```diff
@@ -33,52 +33,59 @@ New features should also be reported as [issues]. Please try to describe the fea
 * WhispersHF
 * Kato
 
+
 #### Senior Moderators
 
 * [0x22]
-* [Scandal]
+* [BongEatBong]
 * [Peanut]
-* SlayerFox
 * [positive]
+* [Scandal]
+* SlayerFox
 
 
 #### Moderators
 
-* [MightyKnight]  
 * Blusk
-* Finlay
-* Seven Oaks
-* [BongEatBong]
-* pepsi boy
-* [valthewyvern]
-* Dectonic
-* ether (Pixel)
 * [CesiumCs]
+* Doctor_Bang
+* elongated muskrat
+* ether (Pixel)
+* Finlay
+* [MightyKnight]
+* [pepsi boy]
 
 
 #### Support
-  
-* sakuya
-* [vitor is]       
-* xpielordx
-* ForkyFork
+
 * Babbaj
+* BeeVa
+* [Berrely]
+* Ezioblade12
+* FakeAccount1231
```

Comment:
wh

Reply:
I just went through everyone with the roles